Gemfile

```diff
@@ -5,6 +5,7 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
+ruby '2.5.7'
```

Comment:
@Ricar7o, while pretending to be a candidate, I tested this locally and encountered the following error:
> nokogiri-1.11.3-x86_64-darwin requires ruby version < 3.1.dev, >= 2.5, which is incompatible with the current version, ruby 2.4.3p205

This additional commit was needed to correct this error.

Reply:
Got it, thanks for catching!